src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -271,6 +277,67 @@ std::vector<std::vector<std::string>> LiveFunctionsDataView::GetContextMenuWithG
   return menu;
 }
 
+ErrorMessageOr<void> LiveFunctionsDataView::ExportAllEventsToCsv(
+    const std::filesystem::path& file_path, const std::vector<int>& item_indices) {
+  ErrorMessageOr<orbit_base::unique_fd> result = orbit_base::OpenFileForWriting(file_path);
+  if (result.has_error()) {
+    return ErrorMessage{absl::StrFormat("Failed to open \"%s\" file: %s", file_path.string(),
+                                        result.error().message())};
+  }
+  const orbit_base::unique_fd& fd = result.value();
+
+  // Write header line
+  constexpr const char* kFieldSeparator = ",";
+  constexpr const char* kLineSeparator = "\r\n";
+  constexpr size_t kNumColumns = 5;
+  const std::array<std::string, kNumColumns> kNames{"Name", "Thread", "Start", "End",
+                                                    "Duration (ns)"};
+  std::string header_line;
+  for (size_t i = 0; i < kNumColumns - 1; ++i) {
+    header_line.append(FormatValueForCsv(kNames[i]));
+    header_line.append(kFieldSeparator);
+  }
+  header_line.append(FormatValueForCsv(kNames[kNumColumns - 1]));
+  header_line.append(kLineSeparator);
+  auto write_result = orbit_base::WriteFully(fd, header_line);
+  if (write_result.has_error()) {
+    return ErrorMessage{absl::StrFormat("Error writing to \"%s\": %s", file_path.string(),
+                                        write_result.error().message())};
+  }
+
+  for (int row : item_indices) {
+    const FunctionInfo& function = GetInstrumentedFunction(row);
+    std::string function_name = orbit_client_data::function_utils::GetDisplayName(function);
+
+    const uint64_t function_id = GetInstrumentedFunctionId(row);
+    std::vector<const TimerInfo*> timers = app_->GetAllFunctionCalls(function_id);
```

Comment:
this here can be a const ref to the vector. 
You could also not take a new timers variable at all, but instead use `app_->GetAllFunctionCalls(function_id)` in the for loop definition directly

Reply:
Changed to use `app_->GetAllFunctionCalls(function_id)` in the for loop definition. Thanks!